### DISCUSSION TITLE
Need clarification on label in name SC 2.5.3

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#3004] [Q&A]
Hi Everyone,

I have a question on label in name SC 2.5.3.

Here visible label 'Modified Date' is not associated with 'All Dates'.
It is a failure of label in name SC 2.5.3?

Current accessible name is: All Dates

![image](https://user-images.githubusercontent.com/115781693/217294665-0f57265d-699c-4a99-a464-a9e71c0d83aa.png)

**Comment by patrickhlauke** [comment]
yes it's a failure
